NuRadioReco/examples/read_full_CoREAS_shower.py

```diff
@@ -0,0 +1,64 @@
+import NuRadioReco.modules.io.coreas.readCoREASShower
+import NuRadioReco.modules.io.eventWriter
+from NuRadioReco.utilities import units
+
+from NuRadioReco.framework.parameters import showerParameters as shP
+
+from NuRadioReco.detector import generic_detector as detector
+
+import numpy as np
+import argparse
+import datetime
+
+# Parse eventfile as argument
+parser = argparse.ArgumentParser(description='NuRadioSim file')
+parser.add_argument('inputfilename', type=str, nargs='*',
+                    help='path to NuRadioMC simulation result')
+
+parser.add_argument('--detectordescription', type=str, nargs='?',
+                    default='../examples/example_data/dummy_detector.json',
+                    help='path to detectordescription')
+
+args = parser.parse_args()
+
+# initialize modules
+readCoREASShower = NuRadioReco.modules.io.coreas.readCoREASShower.readCoREASShower()
+readCoREASShower.begin(args.inputfilename)
+
+eventWriter = NuRadioReco.modules.io.eventWriter.eventWriter()
+output_filename = "Full_CoREAS_shower.nur"
+eventWriter.begin(output_filename)
+
+for iE, event in enumerate(readCoREASShower.run()):
+    print('Event {}'.format(event.get_id()))
+
+    # initialize detector
+    det = detector.GenericDetector(json_filename=args.detectordescription, default_station=102)
+    det.update(datetime.datetime(2011, 11, 11, 11, 11))
+
+    for station in event.get_stations():
+        station.set_station_time(datetime.datetime(2011, 11, 11))
+
+        sim_station = station.get_sim_station()
```

Comment:
I think it would be better to get rid of the simStation.get_position method and instead add the new station directly in the coREAS reader. I created #156 to discuss this.

Reply:
Because we now handle positions in the detector, we should remove the following line. And probably also the `get_position` function from the sim station right?